changelog.json

```diff
@@ -1,5 +1,16 @@
 {
   "releases": {
+    "2.9.4-beta2": [
+      "[Added] Add support for WezTerm on macOS - #12957. Thanks @theodore-s-beers!",
+      "[Fixed] Prevent styles from high-contrast theme from leaking - #12952",
+      "[Fixed] Show \"add repo\" dialog when opening repo from CLI - #12935",
+      "[Fixed] Merge head is correctly detected after merge conflicts resolved externally - #12881",
+      "[Fixed] Removed glitches when diffs are shown for the first time - #12903",
+      "[Improved] Remove gaps in the commit message container revealing a different background - #12900. Thanks @haykam821!",
+      "[Improved] Update \"remove repository\" dialog - #12497. Thanks @fonsp!",
```

Comment:
```suggestion
      "[Improved] Be more specific about what will get removed in the \"remove repository\" dialog - #12497. Thanks @fonsp!",
```

Reply:
@tidy-dev Just curious if there was a reason this suggestion didn't make it. More than happy to take another swing at it, it sure got wordy, but what we've got now isn't great

Reply:
I thought I accepted all of them. Sorry- must have scrolled passed it.